### PR TITLE
[CSM-400] Display logs of wallet tracker

### PR DESCRIPTION
**Pull request by Martoon-00**
This PR is to display most of the logs produced by wallet tracker, in order to facilitate further debugging with user-provided logs.
It also adds some missing info.

### CODE REVIEW

**Comment by gromakovsky** [comment]
CI passed btw. Also reminder: don't forget to make a PR to master after this one is merged.

**Comment by pva701** [comment]
@gromakovsky I took care of similar changes into master. @Martoon-00 can you verify that?

**Comment by gromakovsky** [comment]
It doesn't matter, PR must be made.

**Comment by pva701** [comment]
V sm, ya govoru: v master eto uje est'. uje bil PR kotoriy delal to je samoe v master

**Comment by gromakovsky** [comment]
It doesn't matter, cardano-sl-0.5 must be predecessor of master.
Fortunately, @Martoon-00 got the idea, I think.

**Comment by Martoon-00** [comment]
`merge -s ours` again, fuh

File: src/Pos/Wallet/Web/BListener.hs [hunk 1]
```diff
@@ -82,7 +83,8 @@ onApplyTracking
     , MonadDBRead m
     )
     => OldestFirst NE (Blund ssc) -> m SomeBatchOp
-onApplyTracking blunds = do
+onApplyTracking blunds = setLogger $ do
+    logWarning "This is ololo!"
```
Comment (Martoon-00) [inline src/Pos/Wallet/Web/BListener.hs]: How ever did this pass review

Comment (pva701) [inline src/Pos/Wallet/Web/BListener.hs]: ajajajaj, v golos. I scrolled through PR

**Pull request merged**